eval/templates.py

```diff
@@ -16,6 +16,24 @@ def create_prompt_with_tulu_chat_format(messages, bos="<s>", eos="</s>", add_bos
     formatted_text = bos + formatted_text if add_bos else formatted_text
     return formatted_text
 
+# weirdness with olmo tokenizer means IP_ADDR is the eos token.
+def create_prompt_with_tulu_chat_format_olmo_format(messages, bos="|||IP_ADDRESS|||", eos="|||IP_ADDRESS|||", add_bos=True):
```

Comment:
should we call this `create_prompt_with_olmo_chat_format`?

Reply:
Good point, changed to fit with the other template functions.